plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -31,13 +32,19 @@ public SnykExtendedLanguageClient(ProgressManager pm, Preferences preferences) {
     this.preferences = preferences;
   }
 
-  @JsonNotification(value = "$/hasAuthenticated")
+  @JsonNotification(value = "$/snyk.hasAuthenticated")
   public void hasAuthenticated(HasAuthenticatedParam param) {
     preferences.store(Preferences.AUTH_TOKEN_KEY, param.getToken());
     showAuthenticatedMessage();
     enableSnykViewRunActions();
   }
 
+  @JsonNotification(value = "$/snyk.hasDownloadedCli")
+  public void hasDownloadedCli(HasDownloadedCliParam param) {
```

Comment:
Is this message also sent, if a CLI is already existent, so that `enableSnykViewRunActions()` is also called if no download is needed?